addons/product/models/product_pricelist.py

```diff
@@ -153,7 +153,9 @@ def _compute_price_rule(self, products_qty_partner, date=False, uom_id=False):
             'AND (item.pricelist_id = %s) '
             'AND (item.date_start IS NULL OR item.date_start<=%s) '
             'AND (item.date_end IS NULL OR item.date_end>=%s)'
-            'ORDER BY item.applied_on, item.min_quantity desc, categ.complete_name desc, item.id desc',
+            'AND (item.active = TRUE)'
```

Comment:
@nhomar 
Even if you use another module to add the `active` field this module is not easy to inherit in order to add the `active=True` filter.

If you create a separated module you should copy&paste the 166 lines of this method.
However, if there is a change in the original one you won't be warned and you loose the inherit feature since that you should overwrite it completely.

 - The patch allows you warn if there is a change from Odoo core in this method (conflicts applying the patch).
 - The patch allows you have this method inheritable if there is another place where it is inherited for another feature.